### DISCUSSION TITLE
Exit from a compound state (from compound state to outer "outer" state?

### DISCUSSION THREAD

**Discussion by Laro88** [statelyai/xstate#2868] [Q&A]
I am trying to implement a change from a state inside another state (compound state) but I can't get the visualizer to go "out" from a compound state to a different state outside. Something like stateAlive.stateStressed(eventSchock) -> stateDead 

Is this possible, if so please drop a link / quick example? I am using Boost::StateChart and just discovering a new state playground (with a sweet visualizer) - just specify the unique state you wish to transit to.

**Comment by tivac** [comment]
Have the inner state `send()` an event that the outer state can listen to and transition on?

**Comment by ChrisShank** [comment]
Sounds like you need to use an id to target outside of a compound state https://xstate.js.org/docs/guides/ids.html
